src/rdkafka_partition.c

```diff
@@ -4421,6 +4421,91 @@ int rd_kafka_topic_partition_list_regex_cnt(
 }
 
 
+/**
+ * @brief Match function that returns true if topic is non regex.
```

Comment:
```suggestion
 * @brief Match function that returns true if topic is non a regex.
```

Reply:
Its `non regex` but I am changing it to `not a regex`